README.md

```diff
@@ -19,18 +19,15 @@ This library is a client implementation for use with Apollo Client by using cust
 
 ## Installation
 
-`npm install apollo-link-persisted-queries --save`
+`npm install @apollo/link-persisted-queries --save`
 
 ## Usage
 
 The persisted query link requires using the `http-link`. The easiest way to use them together to to concat them into a single link.
 
 ```js
-import { createPersistedQueryLink } from "apollo-link-persisted-queries";
-import { createHttpLink } from "apollo-link-http";
-import { InMemoryCache } from "apollo-cache-inmemory";
-import ApolloClient from "apollo-client";
-
+import { createPersistedQueryLink } from "@apollo/link-persisted-queries";
```

Comment:
Can we lazy load this link, thus lazy-load the heavy graphql library that is used behind? It's only used for APQ, right? I've noticed that it's even bigger than react-dom. It would be nice to load fast first (and show cache data if available), then improve network performances with link-persisted-queries.

![](https://i.imgur.com/z4cdSvF.png)